src/OpenTelemetry.Resources.Azure/README.md

```diff
@@ -1,5 +1,10 @@
 # Resource Detectors for Azure cloud environments
 
+| Status        |           |
+| ------------- |-----------|
+| Stability     |  [Beta](..\..\Readme.md#beta)|
+| Code Owners   |  [@vishweshbankwar](https://github.com/vishweshbankwar), [@rajkumar-rangaraj](https://github.com/rajkumar-rangaraj)|
```

Comment:
One more thing to note here:
If the code owners are OTel .NET maintainer and/or approver, have a way to indicate that. End users looking at this repo would usually want to know if the cmponent is maintained by repo maintainres/approvers or other individuals.

Reply:
for those cases I was just planning to put `@open-telemetry/dotnet-contrib-maintainers` for code owners.

Reply:
I think it is better to be very explicit about the relation with Otel owned groups.

OTel-DotNet maintainers, approvers
OTel-Dotnet-Contrib maintainers, approvers
OTel-Dotnet-Instrumentation - maintainers, approvers.

eg:
[@vishweshbankwar](https://github.com/vishweshbankwar) OTel-Dotnet Approver, OTel-Dotnet-Contrib Maintainer. 

Does this sound better or this is too much details?

Reply:
Kind of a tricky little box here to fill 🤣 IMO what user cares most about: Is someone actively maintaining the thing? If yes, is it an official thing? Like if it says "AWS" does "AWS" actually own it/support it or was it contributed by some random person? Not sure how best to convey this info 🤔

How about a convention like...

```markdown
| Code Owners | Microsoft-supported (@vishweshbankwar, @rajkumar-rangaraj) |
| Code Owners | CompanyName-supported (@some_user_name)) |
| Code Owners | Community-supported (@some_otel_community_group1, @some_otel_community_group2)) |
| Code Owners | Individual-supported (@some_user_name) |
```

Reply:
I think we should leave it at `@open-telemetry/dotnet-contrib-maintainers`. 

I don't think we can put company name here as that's driven by the support policies outside of OTel.

Reply:
> I think we should leave it at @open-telemetry/dotnet-contrib-maintainers. it's easy to see who the maintainers are.

It is easy, but not every component is owned by contrib maintainers right!

For example, https://github.com/open-telemetry/opentelemetry-dotnet-contrib/tree/main/src/OpenTelemetry.Exporter.Geneva is maintained by a set of people including Reiley. And Reiley is not part of contrib approvers/maintainers.

Lets not block on this for now, and start with something simpler, and later evolve to better convey "who maintains this component".

Reply:
> I don't think we can put company name here as that's driven by the support policies outside of OTel.

Not sure what that means TBH. Why couldn't we put Microsoft on something like GenevaExporter or OneCollectorExporter?

Expanding on what I said above, I don't think list of owners is really _that_ useful to anyone.  Sort of nice, but users probably just want to know one thing: What is the support around this component? That's really what we _should_ have in these READMEs perhaps.

Something like this would be awesome...

| Status        |           |
| ------------- |-----------|
| Stability     |  [Beta](..\..\Readme.md#beta)|
| Support policy | ...???... |
| Code Owners   |  ...list here... |

Consider three examples...

Component A:

| Status        |           |
| ------------- |-----------|
| Stability     |  [Stable](..\..\Readme.md#stable)|
| Support policy | [Supported for production](#support-policy) |
| Code Owners   |  Microsoft (list-of-owners) |

Component B:

| Status        |           |
| ------------- |-----------|
| Stability     |  [Stable](..\..\Readme.md#stable)|
| Support policy | [Limited support](#support-policy) |
| Code Owners   |  list-of-owners |

Component C:

| Status        |           |
| ------------- |-----------|
| Stability     |  [Stable](..\..\Readme.md#stable)|
| Support policy | n/a |
| Code Owners   |  @some-guy |

As a user I would find this type of info really helpful in making decisions about what to use. Doesn't have to be solved on this PR... just providing thoughts!

Reply:
> For example, https://github.com/open-telemetry/opentelemetry-dotnet-contrib/tree/main/src/OpenTelemetry.Exporter.Geneva is maintained by a set of people including Reiley. And Reiley is not part of contrib approvers/maintainers.

I see - for such cases I was going to simply add the owners listed here: https://github.com/open-telemetry/opentelemetry-dotnet-contrib/blob/main/.github/component_owners.yml

Planning to use `@open-telemetry/dotnet-contrib-maintainers` only for components which does not have owners listed atm e.g. https://github.com/open-telemetry/opentelemetry-dotnet-contrib/tree/main/src/OpenTelemetry.Instrumentation.AspNetCore.

Reply:
We can keep this simple, as done in the [collector contrib](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/exporter/azuremonitorexporter#azure-monitor-exporter), and enhance it later on a need basis.

I like this format proposed by Cijo - [@vishweshbankwar](https://github.com/vishweshbankwar) OTel-Dotnet Approver, OTel-Dotnet-Contrib Maintainer

Reply:
had an offline discussion, keeping it `@open-telemetry/dotnet-contrib-maintainers` for now. Will need to revisit the points raised by @CodeBlanch on the support part.